mne/epochs.py

```diff
@@ -1700,7 +1708,7 @@ class EpochsArray(Epochs):
     @verbose
     def __init__(self, data, info, events, tmin=0, event_id=None,
                  reject=None, flat=None, reject_tmin=None,
-                 reject_tmax=None, verbose=None):
+                 reject_tmax=None, baseline=(None, 0), verbose=None):
```

Comment:
+1 for None by default. I would actually apply the baseline if not None... Otherwise I fear discrepancy between data and baseline attribute...